package.json

```diff
@@ -99,6 +99,7 @@
     "prepack": "set -ex; npm run lint; npm run build",
     "pretty": "prettier --write '{,{api,proxy,src}/**/}*.js{,x}'",
     "re": "semantic-release --repository-url git@github.com:solana-labs/blockexplorer.git",
+    "start:uptime-crawler": "set -ex; redis-cli ping; babel-node --presets env api/uptime-crawler.js",
```

Comment:
Any reason this can't be rolled into `start:api`?  That'll be much easier from a integration stand-point (see https://github.com/solana-labs/blockexplorer/blob/master/bin/blockexplorer.sh for what I mean)

Reply:
The main reasons I didn't were:

* it's shelling out to an external program, so more chance of failures to take down API
* getting the N calls away from the API's event loop

If you think it's ok for now, I'll roll it in!

Reply:
Roll it in!